en/docs/how-tos/rate-limiting/adding-throttling-policies.md

```diff
@@ -1,12 +1,20 @@
 # Adding Throttling Policies
```

Comment:
This is only valid for local throttling. Use heading `Define throttle policies for local throttling`